archstrike/empire/PKGBUILD

```diff
@@ -71,7 +71,7 @@ if [[ -f "/usr/share/empire/data/empire.db" ]]; then
 fi
 echo "DB not found continuing with setup"
 cd /usr/share/empire/setup
-./setup_database.py
+python2 setup_database.py
```

Comment:
This change requires that the `pkgrel=2` goes to `pkgrel=3` for users to get the update. So, once that is changed you should be good to go

Reply:
Thanks, made the change